### PR TITLE
feat: isLinear + cliffDuration

### PR DESCRIPTION
**Pull request by fzavalia**

### CODE REVIEW

File: contracts/PeriodicTokenVesting.sol [hunk 1]
```diff
@@ -35,6 +35,9 @@ contract PeriodicTokenVesting is OwnableUpgradeable, PausableUpgradeable {
     /// @dev The duration in seconds of a vesting period.
     uint256 private periodDuration;
```
Comment (nachomazzara) [inline contracts/PeriodicTokenVesting.sol:36]: I would name it `period`

File: contracts/PeriodicTokenVesting.sol [hunk 2]
```diff
@@ -35,6 +35,9 @@ contract PeriodicTokenVesting is OwnableUpgradeable, PausableUpgradeable {
...
 
+    /// @dev The duration in seconds of the cliff.
+    uint256 private cliffDuration;
```
Comment (nachomazzara) [inline contracts/PeriodicTokenVesting.sol:39]: I would name it `cliff`

**Review by nachomazzara — CHANGES_REQUESTED** [review]
small changes request

File: contracts/PeriodicTokenVesting.sol (hunk 1 above)
Comment (fzavalia) [reply contracts/PeriodicTokenVesting.sol:36]: Adding duration seems fitting as the name itself suggests it involves time. 

Calling it only period might be misleading as "period" could be anything, a counter, and index, etc.

File: contracts/PeriodicTokenVesting.sol (hunk 2 above)
Comment (fzavalia) [reply contracts/PeriodicTokenVesting.sol:39]: Same as with period, cliff alone as a uint256 could be the timestamp when the cliff ends (like the old contract) or the duration. 

Appending the duration gives the variable a more precise meaning of what it is used for.

**Pull request merged**